### DISCUSSION TITLE
Additional backup metrics required

### DISCUSSION THREAD

**Discussion by OliverHfm** [cloudnative-pg/cloudnative-pg#1325] [Ideas]
Current capability to monitor status of backup jobs is limited to the first (oldest) point of recoverability (cnp_collector_first_recoverability_point). Additional information like following would be needed:

- Last backup status : Failed / Success
- Last time failed
- Last backup time

The main objective behind the requirement is the ability to provide a minimum level of alerting for backups in a production environment. The above metrics are a base set required to alert DBAs / SREs about proper functioning of the implemented backup jobs.

**Comment by NicoJDE** [comment]
+1 good idea

**Reply by gbartolini** [reply]
My suggestion is to only add one metric that is related to `cnp_collector_first_recoverability_point` and that identifies the timestamp of the end of the last backup. We could call it `cnp_collector_last_available_backup_timestamp`. If you know the frequency of the backups, you can easily set alerts if the backup is older then, for example, 24 hours or so.

**Reply by gbartolini** [reply]
In any case, the information is already present at the Kubernetes level, through the `Backup` resources.

**Comment by gbartolini** [comment]
I have created an issue for this - #1427
